crates/youki/build.rs

```diff
@@ -0,0 +1,8 @@
+use anyhow::Result;
+use vergen::{Config, ShaKind, vergen};
+
+fn main() -> Result<()> {
```

Comment:
Why do we need to use `anyhow::Result` in `build.rs`?

Reply:
The [`vergen` function](https://docs.rs/vergen/latest/vergen/fn.vergen.html) returns a [`Result`](https://docs.rs/anyhow/1.0.51/anyhow/type.Result.html).

I based the code on [this](https://docs.rs/vergen/latest/vergen/#buildrs) example.

If there is another (or better) way to accomplish this, feel free to suggest it :)

Reply:
Oh sorry, my point was ignorant.